File: C++/1020.cpp
#include <stdio.h>

 
int main() {
 int dias, contA, contM;
 contA=0;
 contM=0;
 scanf("%d",&dias);
 
 while(dias>=365){
 	contA=contA+1;
 	dias=dias-365;
 }
 while(dias>=30){
 	contM=contM+1;
 	dias=dias-30;
 }
 printf("%i ano(s)\n",contA);
 printf("%i mes(es)\n",contM);
 printf("%i dia(s)\n",dias);
 return 0;
}

File: Python/1020.py
a = int(input())
cont_anos = 0
cont_meses = 0

while a>=365:
    a = a - 365
    cont_anos= cont_anos+ 1
while a>=30:
    a = a - 30
    cont_meses = cont_meses + 1
print(f'{cont_anos} ano(s)')
print(f'{cont_meses} mes(es)')
print(f'{a} dia(s)')